spec/function.dd

```diff
@@ -2054,14 +2054,12 @@ void func(T)(T x)
 {
     writeln(x);
 }
-
```

Comment:
It looks like we can use macros inside DDoc example blocks, so if it comes to that, you could define a $(EMPTY_LINE) which expands to nothing in .tex and an empty line otherwise.